### ISSUE TITLE
Vagrant File Synced Folder

### ISSUE THREAD

**Issue by selwyni** [cmu-db/ottertune#54]
Vagrant has the capability to sync folders between the host machine and the VM.

This is potentially more secure than symlinks and portable to other hypervisors since Vagrant handles the configurations.

Probably helps deal with the long filenames and development process.

**Comment by dvanaken** [comment]
@selwyni what fix are you suggesting here?

**Comment by selwyni** [comment]
```
 # Customize the amount of memory on the VM:
  config.vm.provider "virtualbox" do |vb|
     vb.memory = "2048"
     vb.name = "OtterTuneVM"
  end

  # Sync to VM
  config.vm.synced_folder "../../../..", "/home/ubuntu/syncedOtterTune"
```
added to the vagrant file - syncedOtterTune might be redundant, or can be named something else.

In this situation, any small change in the ottertune directory is propagated to the VM

**Comment by dvanaken** [comment]
Fixed in #61
